scala/scala.bzl

```diff
@@ -282,6 +319,9 @@ scala_library = rule(
   implementation=_scala_library_impl,
   attrs={
       "main_class": attr.string(),
+      #TODO the implementation of java_library leads me to believe
+      # we may be able to eliminate this
+      "srcjars": attr.label_list(),
```

Comment:
yeah, I think we can get rid of it...I added this before I realized how structs were used to pass around information via their attributes. will tackle this tomorrow